app/components/Views/Settings/NetworksSettings/NetworkSettings/index.js

```diff
@@ -1596,6 +1602,11 @@ export class NetworkSettings extends PureComponent {
     const [, networkConfiguration] = entry;
     const { NetworkController } = Engine.context;
     NetworkController.removeNetwork(networkConfiguration.chainId);
+
+    MetaMetrics.getInstance().addTraitsToUser(
+      getChainIdListPropertyWithout(networkConfiguration.chainId),
```

Comment:
looks like this name needs to be the new one?
```suggestion
      removeItemFromChainIdList(networkConfiguration.chainId),
```